dockers/docker-dhcp-server/cli/show/plugins/show_dhcp_server.py

```diff
@@ -79,5 +79,24 @@ def range(db, range_name):
     click.echo(tabulate(table, headers=headers))
 
 
+@ipv4.command()
+@click.argument('interface', required=False)
```

Comment:
Would it better to set default "*" to `interface`? Then we can remove line 86-87

Reply:
#Closed Sync offline, since it is an issue with all commands, prefer to change them all at once in a separate pr